packages/fungible-token/bridge-fungible-token/proxy/src/proxy.sw

```diff
@@ -0,0 +1,94 @@
+contract;
+
+use std::execution::run_external;
+use standards::{src14::SRC14, src5::{AccessError, State}};
+
+pub enum ProxyErrors {
+    IdentityZero: (),
+}
+
+abi Proxy {
+    #[storage(read)]
+    fn _proxy_owner() -> State;
+
+    #[storage(read)]
+    fn _proxy_target() -> ContractId;
+
+    #[storage(read, write)]
+    fn _proxy_change_owner(new_owner: Identity);
+
+    #[storage(read, write)]
+    fn _proxy_revoke_ownership();
```

Comment:
Have you considered a single function for ownership changes rather than two separate functions? `State::Revoked` would be passed as the function argument.
```sway
    #[storage(read, write)]
    fn _proxy_change_owner(new_owner: State);
```

Reply:
I have, but I thought it would be more idiomatic this way and somewhat harder to mess it up, idk